### PR TITLE
Allow to configure gesture type with TapTool

### LINKED ISSUE #7845: Allow both tap and doubletap gestures with TapTool

### PR DESCRIPTION
**Pull request by mattpap**
fixes #7845

### CODE REVIEW

File: bokehjs/src/lib/core/ui_events.ts [hunk 1]
```diff
@@ -403,10 +404,7 @@ export class UIEvents {
   }
 
   protected _doubletap(e: HammerEvent): void {
-    // NOTE: doubletap event triggered unconditionally
```
[the remark this answers is not in the document]
Comment (mattpap) [reply bokehjs/src/lib/core/ui_events.ts]: This breaks poly select tool

[the remark this answers is not in the document]
Comment (mattpap) [reply bokehjs/src/lib/core/ui_events.ts]: In fact this breaks all tools depending on `_doubletap` and toolbar management in general (active tools).

[the remark this answers is not in the document]
Comment (bryevdv) [reply bokehjs/src/lib/core/ui_events.ts]: Probably me, sorry. I don't recall the details, only that there was something weird and annoying about how hammer.js handled double tap (i.e. I think maybe I was having trouble getting what should have the the obvious API usage to correctly differentiate events, or not fire extra events, or something)

**Comment by mattpap** [comment]
At this point I think that ui_events needs a proper redesign to allow such case and to be easier to work with in general. I will save this branch for later user.

**Pull request closed without merging**